modules/claims_api/config/initializers/okcomputer.rb

```diff
@@ -96,11 +123,22 @@ def name
 OkComputer::Registry.register 'mpi', MpiCheck.new
 OkComputer::Registry.register 'bgs-vet_record', BgsCheck.new('vet_record')
 OkComputer::Registry.register 'bgs-corporate_update', BgsCheck.new('corporate_update')
-OkComputer::Registry.register 'bgs-intent_to_file', BgsCheck.new('intent_to_file')
-OkComputer::Registry.register 'bgs-claimant', BgsCheck.new('claimant')
 OkComputer::Registry.register 'bgs-contention', BgsCheck.new('contention')
-OkComputer::Registry.register 'bgs-ebenefits_benefit_claims_status', BgsCheck.new('ebenefits_benefit_claims_status')
-OkComputer::Registry.register 'bgs-tracked_items', BgsCheck.new('tracked_items')
+
+OkComputer::Registry.register 'localbgs-find_poa_by_participant_id',
+                              FaradayBGSCheck.new('find_poa_by_participant_id', '0')
+OkComputer::Registry.register 'localbgs-find_by_ssn', FaradayBGSCheck.new('find_by_ssn', '0')
+OkComputer::Registry.register 'localbgs-find_poa_history_by_ptcpnt_id',
+                              FaradayBGSCheck.new('find_poa_history_by_ptcpnt_id', '0')
+OkComputer::Registry.register 'localbgs-find_benefit_claims_status_by_ptcpnt_id',
+                              FaradayBGSCheck.new('find_benefit_claims_status_by_ptcpnt_id', '0')
+OkComputer::Registry.register 'localbgs-find_benefit_claim_details_by_benefit_claim_id',
+                              FaradayBGSCheck.new('find_benefit_claim_details_by_benefit_claim_id', '0')
```

Comment:
I am curious why some of the `local_bgs` services are "unavailable", is it the services without a key?

Reply:
Good catch! A few of the local_bgs methods were throwing exceptions when supplied a dummy/default parameter:
find_benefit_claim_details_by_benefit_claim_id
Common::Exceptions::ServiceError Exception: Unknown Service Error
find_tracked_items
Common::Exceptions::ResourceNotFound Exception: Resource not found’
find_benefit_claims_status_by_ptcpnt_id
Common::Exceptions::ServiceError Exception: Unknown Service Error

I solved this in the latest push by supplying parameters that work in sandbox & staging, but I'm guessing they will fail in other environments. Not sure how to get around this other than supplying specific parameters for things like claim_id and participant_id used by the services?

Reply:
We could use the `@env` variable, found [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/lib/bgs_service/local_bgs.rb), or something like it, and based on the that variable switch the ids.